versioned_docs/version-latest/protocol/substrate-cli.mdx

```diff
@@ -606,209 +583,221 @@ Download the Executable Files, using the appropriate commands:
     </summary>
 		Node:
 		<CodeBlock language="shell-session">
-		wget -O /root/.local/bin/subspace-node https://github.com/subspace/subspace/releases/download/gemini-3g-2023-sep-25/subspace-node-ubuntu-aarch64-gemini-3g-2023-sep-25
+		wget -O subspace-node https://github.com/subspace/subspace/releases/download/gemini-3g-2023-nov-07/subspace-node-ubuntu-aarch64-gemini-3g-2023-nov-07
 		</CodeBlock>
 		Farmer:
 		<CodeBlock language="shell-session">
-		wget -O /root/.local/bin/subspace-farmer https://github.com/subspace/subspace/releases/download/gemini-3g-2023-sep-25/subspace-farmer-ubuntu-aarch64-gemini-3g-2023-sep-25
+		wget -O subspace-farmer https://github.com/subspace/subspace/releases/download/gemini-3g-2023-nov-07/subspace-farmer-ubuntu-aarch64-gemini-3g-2023-nov-07
 		</CodeBlock>
 </details>
 
-By default, these commands download the executable files to the `/root/.local/bin` directory. 
-You can change the directory by modifying the respective string.
-
-You can use the same commands to update the node.
-
-Make the Files Executable:
-```shell-session
-chmod +x /root/.local/bin/subspace-node
-chmod +x /root/.local/bin/subspace-farmer
+Copy them to `/usr/local/bin` and delete originals:  
+```console
+sudo cp subspace-node subspace-farmer /usr/local/bin
 ```
+Make them executable:
+```console
+sudo chmod +x /usr/local/bin/subspace-node /usr/local/bin/subspace-farmer
+```
+*However, you can download binaries directly to the right place if you run these commands from `/usr/local/bin` directory with root privilegies.*
 
-Create a Directory for Node and Farmer Data:
-```shell-session
-mkdir -p /root/.local/share
+Set ownership to user 'subspace':
+```console
+sudo chown subspace:subspace /usr/local/bin/subspace-node
+sudo chown subspace:subspace /usr/local/bin/subspace-farmer
```

Comment:
(Just because they are in a folder accessible to all users, I made the owner of those files -subspace so that other users can't change them, only the subspace itself. )
    But you're right that in their own home directory they won't need to set additional permissions and will even be more organized. And yes, sort of without sacrificing security.